src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private class Node<T> {
```

Comment:
Usually private nested classes (along with their methods) are placed after the upper-class methods, so let's move the Node class to the end